src/pages/style.css

```diff
@@ -81,6 +81,29 @@ button {
 
 /* General helper classes */
 
+/*
+TODO: this will eventually replace the other focus helpers but they are heavily
+used at the moment and need to be refactored carefully. The places they are used
+do extra math to account for padding, which this helper now does with padding
+and negative margin.
+*/
+.focus-ring-from-box-shadow {
```

Comment:
non actionable thought 💭 : I wonder if we might want to prefix these global helper classes with `h-`, or `g-` like our other global classes, to make it explicit during usage.

Reply:
What does the `h-` stand for? I assume `g-` stands for global 😁

Reply:
`h-` would stand for "helper"!

Reply:
That makes total sense! I think this is a good idea, will keep it in mind and try to add it soon. Maybe when the old ones get refactored out is a good opportunity to do it.